src/panels/usesSummaryPanel/UseSummaryPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import { Card, CardContent, CardHeader } from "@material-ui/core";
+import { Activities, Environments, IUse } from "entities/IUse";
+import React, { FunctionComponent, ReactNode } from "react";
+import { AviationSummary } from "./AviationSummary";
+import { LandSummary } from "./LandSummary";
+import { MaritimeSummary } from "./MaritimeSummary";
+
+interface UseSummaryPanelProps {
+  use: IUse;
+  titlePrefix: string;
+}
+
+export const UseSummaryPanel: FunctionComponent<UseSummaryPanelProps> = ({
+  use,
+  titlePrefix,
+}: UseSummaryPanelProps): JSX.Element => {
+  const title = getCardHeaderTitle(titlePrefix, use);
+  const useSummary = getUseSummary(use);
```

Comment:
Is prefixing every function here with `get` necessary?  `cardHeaderTitle()` and `useSummary()` work fine for me.  This is very Javaland-y 😝